sdk/ai/Azure.AI.Inference/tests/Utilities/ValidatingActivityListener.cs

```diff
@@ -0,0 +1,208 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text.Json;
+using Azure.AI.Inference.Telemetry;
+using NUnit.Framework;
+
+using static Azure.AI.Inference.Telemetry.OpenTelemetryConstants;
+
+namespace Azure.AI.Inference.Tests.Utilities
+{
+    [NonParallelizable]
+    internal class ValidatingActivityListener : IDisposable
+    {
+        private readonly ActivityListener m_activityListener;
+        private readonly ConcurrentQueue<Activity> m_listeners = new();
+
+        private void activityStopped(Activity act)
```

Comment:
Please follow repository code styles and use PascalCase for all methods.

Reply:
Fixed